en/docs/fragments/set-mandatory-attributes.md

```diff
@@ -0,0 +1,3 @@
+To make the attributes configured above appear for the user to fill in or consent to during login, enable **Requested** and **Mandatory** for these attributes.  
```

Comment:
To prompt users to fill in or consent to the attributes configured above, enable....